lib/matplotlib/tests/test_axes.py

```diff
@@ -6824,3 +6826,15 @@ def test_ylabel_ha_with_position(ha):
     ax.set_ylabel("test", y=1, ha=ha)
     ax.yaxis.set_label_position("right")
     assert ax.yaxis.get_label().get_ha() == ha
+
+
+def test_patch_bounds():  # PR 19078
+    fig, ax = plt.subplots()
+    tol = 16*sys.float_info.epsilon
+    ax.add_patch(mpatches.Wedge((0, -1), 1.05, 60, 120, 0.1))
+    bounds = ax.dataLim.bounds
+    bot = 1.9*math.sin(15*math.pi/180)**2
+    assert abs(bounds[0]+0.525) < tol and \
+           abs(bounds[1]+(bot+0.05)) < tol and \
+           abs(bounds[2]-1.05) < tol and \
+           abs(bounds[3]-(bot+0.1)) < tol
```

Comment:
Can use something from `np.testing.assert_*` instead of a manual tolerance.

Reply:
Done